### ISSUE TITLE
Load actor avatar when user presses "join world" button

### ISSUE THREAD

**Issue by shawticus** [xr3ngine/xr3ngine#620]
Right now we are instantiating the avatar for the user when they connect to the world.

However, what we would like to do is position the camera at the "scene preview camera" object position and rotation (this is set in the editor), then when they press the "join world" UI button they will receive an avatar and the camera will lerp to this position.

**Comment by XiaNi** [comment]
@tanya-vykliuk attach please "join world" code to button.
code is located at `packages/client/pages/location/[locationName].tsx`, and it looks like:
```
    console.log('!! --- JOIN in 10 sec');
    setTimeout(async () => {
      console.log('!! --- JOIN now!');
      console.log("Joining world");

      // TODO: make it proper way!?
      const joinWorldResponse = await (Network.instance.transport as SocketWebRTCClientTransport).request(MessageTypes.JoinWorld.toString());
      const { worldState } = joinWorldResponse as any;
      console.log("Joined world");
      // Apply all state to initial frame
      console.log("Apply state?", worldState);
      applyNetworkStateToClient(worldState);
    }, 10000);
```

all you need from it is this small part:
```
      const joinWorldResponse = await (Network.instance.transport as SocketWebRTCClientTransport).request(MessageTypes.JoinWorld.toString());
      const { worldState } = joinWorldResponse as any;
      applyNetworkStateToClient(worldState);
```

**Comment by XiaNi** [comment]
closed with #824
